content/docs/use-cases/sharing-resources.md

```diff
@@ -0,0 +1,54 @@
+# Sharing Resources
+
+DVC enables different ways to optimize resource utilization for scenarios where
```

Comment:
let's for now try to start w/o mentioning DVC. It should be about the problem/use case in general first. 

e.g. Team -> lots of data -> run a lot of training -> manage data effificiently, distribute GPUs, etc.

Reply:
I updated the intro (and added a first section on storage). PTAL